gato/policy/gato_policy.py

```diff
@@ -89,6 +91,7 @@ def __init__(
                 config=config,
             )
             embed_dim = config.n_embd
+            self.embed_token = self.transformer.wte
```

Comment:
I want to modify this line. The size of self.transformer.wte should be equal to text_tokens, but you want to expand your nn.Embeddding to go up to vocab_size. I can work on modifying this, following from here https://github.com/machelreid/can-wikipedia-help-offline-rl/blob/e1060f0e03a22e3c05fb70cbad05dfc840ce8afc/code/decision_transformer/models/decision_transformer.py#L78

Reply:
Go for it! Thanks for the support. :)